readthedocs/search/tests/data/generic/out/basic.json

```diff
@@ -1,11 +1,46 @@
 [
   {
-    "title": "Title of the page",
     "path": "basic.html",
+    "title": "Title of the page",
     "sections": [
+      {
+        "id": "love",
+        "title": "Love",
+        "content": "To code is to love"
+      },
+      {
+        "id": "code",
+        "title": "Code",
+        "content": "Code and docs are like love, they are 4-letter words."
+      },
+      {
+        "id": "docs",
+        "title": "Docs",
+        "content": "Code and docs are like love, they are 4-letter words."
+      },
+      {
+        "id": "complexity",
+        "title": "Complexity",
+        "content": ""
+      },
+      {
+        "id": "mother",
+        "title": "Mother",
+        "content": "Mother cannot fly"
+      },
+      {
+        "id": "stones",
+        "title": "All stones",
+        "content": "All stones cannot fly"
+      },
+      {
+        "id": "ergo",
+        "title": "Ergo",
+        "content": "Mother is a stone"
+      },
       {
         "id": "",
-        "title": "Title of the page",
+        "title": "Ergo",
```

Comment:
Oh this is actually quite strange, it looks like a variable from a previous loop is spilling over

Reply:
Fixed